time-series/time-series-api/src/test/java/com/powsybl/timeseries/CalculatedTimeSeriesTest.java

```diff
@@ -84,21 +86,33 @@ public Set<Integer> getTimeSeriesDataVersions(String timeSeriesName) {
 
             @Override
             public List<DoubleTimeSeries> getDoubleTimeSeries(Set<String> timeSeriesNames) {
-                List<DoubleTimeSeries> timeSeriesList = new ArrayList<>(1);
+                List<DoubleTimeSeries> timeSeriesList = new ArrayList<>(2);
                 if (timeSeriesNames.contains("ts")) {
                     timeSeriesList.add(ts);
                 }
+                if (timeSeriesNames.contains("foo")) {
+                    timeSeriesList.add(foo);
+                }
                 return timeSeriesList;
             }
         };
-        CalculatedTimeSeries tsCalc = new CalculatedTimeSeries("ts_calc", BinaryOperation.plus(new TimeSeriesNameNodeCalc("ts"),
-                                                                                               new IntegerNodeCalc(1)));
+        CalculatedTimeSeries tsCalc = new CalculatedTimeSeries("ts_calc", BinaryOperation.plus(
+            new TimeSeriesNameNodeCalc("ts"),
+            new IntegerNodeCalc(1)));
+        CalculatedTimeSeries tsCalcMin = new CalculatedTimeSeries("ts_calc_min", new BinaryMinCalc(
+            new TimeSeriesNameNodeCalc("ts"),
+            new TimeSeriesNameNodeCalc("foo")));
+        CalculatedTimeSeries tsCalcMax = new CalculatedTimeSeries("ts_calc_max", new BinaryMaxCalc(
+            new TimeSeriesNameNodeCalc("ts"),
+            new TimeSeriesNameNodeCalc("foo")));
 
         // check versions of the data available for the calculated time series
         tsCalc.setTimeSeriesNameResolver(resolver);
+        tsCalcMin.setTimeSeriesNameResolver(resolver);
+        tsCalcMax.setTimeSeriesNameResolver(resolver);
         assertEquals(Sets.newHashSet(1), tsCalc.getVersions());
 
-        List<DoubleTimeSeries> tsLs = Arrays.asList(ts, tsCalc);
+        List<DoubleTimeSeries> tsLs = Arrays.asList(ts, tsCalc, tsCalcMin, tsCalcMax);
         String json = TimeSeries.toJson(tsLs);
         String jsonRef = String.join(System.lineSeparator(),
```

Comment:
Now that we have text blocks in Java 17, we can do it much simpler and cleaner.
Maybe should be done in another PR.